test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/GoodsRouteDestinationPageSpec.scala

```diff
@@ -45,5 +45,5 @@ class GoodsRouteDestinationPageSpec extends DeclarationDataCapturePageSpec[YesNo
   // PH - I think this should be persisted on the DeclarationJourney
```

Comment:
the comments could be removed as part of this ticket

Reply:
will clean up 🧹 no problems, Thanks Paul